### PR TITLE
Move the RNR forms to Replenishment menu 

### LINKED ISSUE #4520: Move R&R to Requisitions menu
## Is your feature request related to a problem? Please describe 👀
Sorry this has come so late - was meaning to canvas opinions earlier!

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #4520

# 👩🏻‍💻 What does this PR do?
By popular request, moves to `Replenishment`
I've moved the files to `requisitions` rather than leave them under `programs` which explains the number of files. 
It's all just shuffling around though.

## 💌 Any notes for the reviewer?

# 🧪 Testing

- [ ] Does it look ok? Can you still view RnR?

# 📃 Documentation

- [x] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: client/packages/requisitions/src/RnRForms/ListView/ProgramSearchInput.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from 'react';
 import { Autocomplete } from '@openmsupply-client/common';
-import { ProgramFragment } from '../../api/operations.generated';
-import { useProgramList } from '../../api';
+import { ProgramFragment } from '../api/operations.generated';
+import { useProgramList } from '../api';
 
 type ProgramSearchInputProps = {
```
Comment (lache-melvin) [inline client/packages/requisitions/src/RnRForms/ListView/ProgramSearchInput.tsx:6]: Bit of a nit, seeing as this is only used by the R&R forms, but I wonder if this should live in programs still? Or system?

We have the programs list query defined in both the program and r&r operations files now, which I think is out of line with existing patterns?

File: client/packages/requisitions/src/RnRForms/ListView/RnRFormCreateModal.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ import {
 import { AppRoute } from '@openmsupply-client/config';
 import { SupplierSearchInput } from '@openmsupply-client/system';
 import { ProgramSearchInput } from './ProgramSearchInput';
-import { useCreateRnRForm, useSchedulesAndPeriods } from '../../api';
+import { useCreateRnRForm, useSchedulesAndPeriods } from '../api';
```
Comment (lache-melvin) [inline client/packages/requisitions/src/RnRForms/ListView/RnRFormCreateModal.tsx:19]: Just need to update the post-create navigate call! 

Screenshot 2024-08-07 at 5 19 56 PM

**Review by lache-melvin — CHANGES_REQUESTED** [review]
Looks good to me, thanks Mark! Just the redirect to fix, and maybe where the programs query lives?

I'll update the docs screenshots with these changes in the morning 😁

File: client/packages/requisitions/src/RnRForms/ListView/ProgramSearchInput.tsx (hunk 1 above)
Comment (mark-prins) [reply client/packages/requisitions/src/RnRForms/ListView/ProgramSearchInput.tsx:6]: true, will consolidate 🙏

File: client/packages/requisitions/src/RnRForms/ListView/RnRFormCreateModal.tsx (hunk 2 above)
Comment (mark-prins) [reply client/packages/requisitions/src/RnRForms/ListView/RnRFormCreateModal.tsx:19]: yep. had done - just in the second commit.. so a bit too slow!

**Review by lache-melvin — APPROVED** [review]
Very nice, thanks Mark 🚀  Just a couple of translation fixes, merge when ready 😁

File: client/packages/common/src/intl/locales/en/programs.json [hunk 3]
```diff
@@ -11,14 +10,10 @@
...
+  "description.expiry": "The closest expiry date of current stock on hand",
   "label.bmi-prev-height-message": "Using height of {{height}}m from previous measurement",
   "label.diastolic": "Diastolic",
-  "label.new-rnr-form": "New R&R Form",
-  "label.schedule": "Schedule",
   "label.systolic": "Systolic",
-  "messages.click-to-return-to-rnr-list": "Click here to return to the list of R&R Forms",
   "messages.all-lines-must-be-confirmed": "All lines must be confirmed before the R&R form can be finalised",
   "messages.finalise-previous-form": "Finalise the previous R&R form in this schedule to create the next one",
   "messages.no-available-periods": "No available periods found for schedule"
```
[the remark this answers is not in the document]
Comment (mark-prins) [reply client/packages/common/src/intl/locales/en/programs.json:19]: thanks! that's the one I spotted last night. I knew that moving these translations would be risky 🙄

File: client/packages/common/src/intl/locales/ru/programs.json [hunk 4]
```diff
@@ -1,25 +1,22 @@
...
+  "control.adherence-status-warning": "Количество Таблеток: после последнего посещения = {{previousCountOnHand}} < перед посещением = {{remainingCount}}",
+  "control.search.no-results-label": "Нет результатов",
+  "control.search.searching-label": "Поиск...",
+  "control.search.reset-button": "Отвязать сопоставленного пациента",
+  "label.diastolic": "Диастолическое",
+  "label.systolic": "Систолическое",
+  "label.bmi-prev-height-message": "Используя высоту {{height}}м из предыдущего измерения",
+  "error.no-rnr-forms": "О&З Формы не найдены",
+  "label.new-rnr-form": "Новая О&З Форма",
+  "error.rnr-not-found": "О&З Форма не найдена",
```
[the remark this answers is not in the document]
Comment (mark-prins) [reply client/packages/common/src/intl/locales/ru/programs.json:18]: we used to have them go to main - and I changed to develop to reduce some merge issues around release time..

**Pull request merged**